mealy/error_tree.py

```diff
@@ -84,4 +84,4 @@ def _compute_ranking_arrays(self, n_purity_levels=ErrorAnalyzerConstants.NUMBER_
 
         #y = self._error_train_y
         n_total_errors = np.sum(wrongly_predicted_samples) #y[y == ErrorAnalyzerConstants.WRONG_PREDICTION].shape[0]
-        self._global_error = wrongly_predicted_samples.astype(float) / n_total_errors
\ No newline at end of file
+        self._total_error_fraction = wrongly_predicted_samples.astype(float) / n_total_errors
```

Comment:
Easier to cast one iinteger to float than a whole array
```suggestion
        self._total_error_fraction = wrongly_predicted_samples / float(n_total_errors)

```